client/my-sites/checkout/checkout/composite-checkout.js

```diff
@@ -194,6 +201,119 @@ export default function CompositeCheckout( {
 		);
 	};
 
+    function getTermText( term ) {
+        switch ( term ) {
+            case TERM_BIENNIALLY:
+                return translate( 'Two years', {
+                    context: 'subscription length',
+                } );
+
+            case TERM_ANNUALLY:
+                return translate( 'One year', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'One year' )` *ES Score: 12*


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.